federation/manifests/federation-apiserver-deployment.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: extensions/v1beta1
```

Comment:
I'm confused why this change is here?

Reply:
Seems like it's been deleted. Mistakenly added in rebase. Will remove